docs/content/preview/yugabyte-cloud/cloud-connect/connect-applications.md

```diff
@@ -66,15 +66,15 @@ To get the connection parameters for your cluster:
 1. Click **Connect to your Application**.
 1. Click **Download CA Cert** and install the cluster certificate on the computer running the application.
 1. Choose the API used by your application, **YSQL** or **YCQL**, to display the corresponding connection parameters.
-1. If your cluster is deployed in a VPC, choose **Private Address** if your application is in a peered VPC. Otherwise, choose Public Address (only available if you have enabled [Public Access](#vpc-network); not recommended for production).
+1. If your cluster is deployed in a VPC, choose **Private Address** if your application is in a peered VPC. Choose **Private Service Endpoint** if your application is in a linked VPC. Otherwise, choose Public Address (only available if you have enabled [Public Access](#vpc-network); not recommended for production).
```

Comment:
```suggestion
1. If your cluster is deployed in a VPC, choose **Private Address** if your application is in a peered VPC. Choose **Private Service Endpoint** if your application is in a linked VPC. Otherwise, choose Public Address (only available if you have enabled [Public Access](../../../yugabyte-cloud/cloud-secure-clusters/add-connections/#enabling-public-access); not recommended for production).
```